api/app/Jobs/GenerateDepictsQuestions.php

```diff
@@ -315,6 +314,7 @@ private function processFilePage( PageIdentifier $filePageIdentifier ) : bool {
                     'old_depicts_name' => $lessSpecificItemLabel->getText(),
                     'depicts_id' => $this->depictItemId,
                     'depicts_name' => $this->depictName,
+                    'img_title' => $filePageIdentifier->getTitle()->getText(),
```

Comment:
The one downside of now adding the page title to the question is that this view is now broken for old questions that have been generated if they were want to get asked again (which I might do).

It's possible to use a link such as https://commons.wikimedia.org/wiki/Special:EntityData/M22518397 which only mkes use of the media info ID to show the file page, so I would propose that we do that!

(I'll make the change and merge this PR! :)

Reply:
I am not surprised that there was a better solution than the naive one I proposed here :) this was just something I whipped up in a few minutes after looking around the codebase for a bit, but without exploring it in depth. Thanks for improving the PR!